package.json

```diff
@@ -5,8 +5,14 @@
   "description": "Module for consuming paginated Django REST framework endpoints.",
   "homepage": "https://github.com/yola/drf-paginator",
   "license": "MIT",
-  "directories": {},
-  "scripts": {},
+  "main": "src/index.js",
+  "scripts": {
+    "cover": "rm -rf coverage && babel-node node_modules/.bin/isparta cover --root src/ --report text --report lcov --report html node_modules/.bin/_mocha -- src/test-setup.js $(find src -name '*-spec.js')",
+    "coveralls": "cat coverage/lcov.info | coveralls",
+    "jshint": "jshint src",
+    "test": "npm run cover && npm run jshint",
```

Comment:
I don't think we want linting to break tests. Snitch does that for us, and because linting errors are often stylelistic, they shouldn't affect tests.

Do you agree?